lib/creatubbles/creation.rb

```diff
@@ -28,6 +26,12 @@ class Creatubbles::Creation < Creatubbles::BaseObject
     updated_at
   ]
 
+  RELATIONSHIP_LIST = %w[user creators]
+
+  define_type_name 'creations'
+  define_attributes ATTRIBUTE_LIST
```

Comment:
any reason to use a constant here? IMHO it doesn't improve the code but just makes it a bit harder to read.